tests/Tests.FeatureManagement/Tests.FeatureManagement.csproj

```diff
@@ -18,13 +18,28 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.5" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.5" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
```

Comment:
Are there any build issues when using version `16.6.1` or newer for `netcoreapp2.1`?

Reply:
Not build issues, but there were runtime issues. The test wouldn't even start. I looked back at what we were using as a reference when we were using netcoreapp2.1 as the test runtime and it was indeed 15.9.0. When I reverted the reference to that for netcoreapp2.1 it worked correctly.